.circleci/config.yml

```diff
@@ -60,6 +87,33 @@ jobs:
           COMPOSE_FILE: ingestion-edge/docker-compose.yml
           COMPOSE_PROJECT_NAME: circleci-ingestion-edge
 
+  ingestion-edge-deploy:
```

Comment:
I think we should use an early return like [jeff's recent article](https://circleci.com/blog/managing-secrets-when-you-have-pull-requests-from-outside-contributors/), docker-compose to push the image, and not split the jobs because it makes the local cli more troublesome. something like:

in `ingestion-edge/docker-compose.yml`:
<pre>
  web:
    image: &image "${DOCKERHUB_REPO-ingestion-edge}:${CIRCLE_TAG-latest}"
</pre>

in `.circleci/config.yml`
<pre>
    - run:
        name: Early return for no deploy
        command: |
                if [ -n "$CIRCLE_PR_NUMBER" ] || ([ "${CIRCLE_BRANCH}" != "master" ] && [ -z "${CIRCLE_TAG}" ]); then
                  echo "Nothing to deploy"
                  circleci step halt
                fi
    - run:
        name: Deploy to Dockerhub
        command: |
          echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
          docker-compose push web
</pre>